dotCMS/src/main/java/com/dotcms/vanityurl/business/VanityUrlAPI.java

```diff
@@ -1,73 +1,79 @@
 package com.dotcms.vanityurl.business;
 
+import com.dotcms.keyvalue.model.KeyValue;
+import java.util.List;
+import java.util.Optional;
+
 import com.dotcms.vanityurl.model.CachedVanityUrl;
 import com.dotcms.vanityurl.model.VanityUrl;
 import com.dotmarketing.beans.Host;
+import com.dotmarketing.exception.DotDataException;
 import com.dotmarketing.portlets.contentlet.model.Contentlet;
-import com.liferay.portal.model.User;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
 
 /**
- * This API provides access to the information related to Vanity URLs
- * in dotCMS. Vanity URLs are alternate reference paths to
- * internal or external URL's. Vanity URLs are most commonly used to give
- * visitors to the website a more user-friendly or memorable way of reaching an
- * HTML page or File, that might actually live “buried” in a much deeper path.
+ * This API provides access to the information related to Vanity URLs in dotCMS. Vanity URLs are
+ * alternate reference paths to internal or external URL's. Vanity URLs are most commonly used to
+ * give visitors to the website a more user-friendly or memorable way of reaching an HTML page or
+ * File, that might actually live “buried” in a much deeper path.
  *
  * @author oswaldogallango
  * @version 4.2.0
  * @since June 12, 2017
  */
 public interface VanityUrlAPI {
+    public static final String DEFAULT_VANITY_URL_STRUCTURE_VARNAME = "Vanityurl";
```

Comment:
Codacy found an issue: [Unnecessary modifiers 'public static final' on field 'DEFAULT_VANITY_URL_STRUCTURE_VARNAME': the field is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)